cms/envs/devstack.py

```diff
@@ -38,10 +38,12 @@
 
 # Revert to the default set of finders as we don't want the production pipeline
 STATICFILES_FINDERS = [
-    'staticfiles.finders.FileSystemFinder',
-    'staticfiles.finders.AppDirectoriesFinder',
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
 ]
 
+PIPELINE_ENABLED = False
```

Comment:
Nit: I'd actually put this first in this section directly beneath the section comment, because it explains all the other settings.